<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addInfo</class>
 <widget class="QDialog" name="addInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="firstLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>First name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="holidayLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vehicleLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mobileLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobile number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="holidayLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Holiday duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vehicleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vehicle registration number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer Profile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mobileLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>exitapp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exitapp()</slot>
 </slots>
</ui>
